psalm.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <psalm
     errorLevel="4"
-    phpVersion="8.0"
+    phpVersion="8.2"
```

Comment:
Please drop this line, then Psalm will inflect the PHP version from `composer.json`